app/services/analytics_events.rb

```diff
@@ -1007,6 +1007,12 @@ def idv_gpo_confirm_start_over_visited
     track_event('IdV: gpo confirm start over visited')
   end
 
+  # A GPO reminder email was sent to the user
+  # @param [String] to_user UUID of user who we sent a reminder to
+  def idv_gpo_reminder_email_sent(to_user:)
+    track_event('IdV: gpo reminder email sent', to_user: to_user)
```

Comment:
We should either:
1. instantiate an `Analytics` instance with the 
2. rename this to `user_id` so that it gets treated as the user_id in the standard spot: https://github.com/18F/identity-idp/blob/0579d710766d1c488234f7a1601d5c74ddd42615/app/services/analytics.rb#L25